packages/data-portal-explore/src/lib/dataTableHelpers.tsx

```diff
@@ -147,8 +147,7 @@ export function truncatedTableCell<T>(cellData: T) {
     const value = _.isFunction(selector)
         ? selector(cellData)
         : cellData[selector as keyof T];
-
     return value ? (
-        <ExpandableText fullText={value} truncateProps={{ lines: 4 }} />
+        <ExpandableText fullText={value} truncateProps={{ lines: 4 }} value={value}/>
```

Comment:
It's a bit confusing to pass the same value as two different props. We should probably come up with a better solution.